extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -230,44 +232,52 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 		).component();
 	}
 
-	private async handleRegistrationsUpdated(): Promise<void> {
+	private handleRegistrationsUpdated(): void {
 		const reg = this._controllerModel.getRegistration(ResourceType.sqlManagedInstances, this._miaaModel.info.namespace, this._miaaModel.info.name);
 		if (reg) {
 			this._instanceProperties.resourceGroup = reg.resourceGroupName || '-';
 			this._instanceProperties.dataController = this._controllerModel.controllerRegistration?.instanceName || '-';
-			this._instanceProperties.region = (await getAzurecoreApi()).getRegionDisplayName(reg.location);
```

Comment:
@Charles-Gagnon I had to move this async call to the model.  The data needs to be available when we're creating the components, but the container() method is not async.

Reply:
Saves having to do multiple mappings anyways so looks good to me